### ISSUE TITLE
Resolve "autocomplete has valid value"

### ISSUE THREAD

**Issue by nitedog** [w3c/wcag-act#501]
This rule checks that "autocomplete has valid value":
[Survey results](https://www.w3.org/2002/09/wbs/93339/ACTAutocomplete/results)

The survey discussed during:
[12 November 2020 meeting](https://www.w3.org/2020/11/12-wcag-act-minutes.html#item01)

**Kathy Eng:**

- [ ] Not clear on second assumption - it seems form fields that do not collect information about the user shouldn't or wouldn't have autocomplete. Is this assumption necessary given the first assumption that when autocomplete is present, it is collecting info about the user?
- [ ] Passed 2, 3, 5, 6, 7 examples need more visible label info to confirm that the optional tokens in the the attribute values are correct.

**Mary Jo Mueller:**

- [ ] Seems the 2nd paragraph in the assumptions indicate it is assumed this rule will produce false positives. For the last paragraph, the last sentence is not super-clear to me. If you qualify "If this is not the case" it would be clearer. I can't tell if the statement is indicating "If aria-disabled state is not used" or "if aria-disabled is used but the element is still operable". When you have a complex statement followed by a statement using "this" it is hard to parse if all of the things from the previous statement need to be true or some.
- [ ] Seems like Passed example 8 should be inapplicable since the widget role is no longer an input - it is overridden by ARIA but not as an input, select or textarea.

** Wilco Fiers:**

- [ ] The rule does not account for different types of disabled elements. Only the aria-disabled attribute. Inapplicable example 6 seems like it should be a pass instead.
- [ ] I'm not too keen on passed example 8. Can we find a better role for this?

**Comment by WilcoFiers** [comment]
PR approved, merging today, should be ready for another review: https://github.com/act-rules/act-rules.github.io/pull/1534

**Comment by WilcoFiers** [comment]
Has 2 open issues to resolve.
